.env

```
@@ -54,6 +54,7 @@ UBUNTU=22.04
 
 # Default versions for various dependencies
 CLANG_TOOLS=14
+CMAKE_VERSION=3.25.0
```

Could you remove `_VERSION` suffix like other variables?

```suggestion
CMAKE=3.25.0
```

---

I made the change from `CMAKE` to `CMAKE_VERSION` because it was requested previously. Not sure if it was @nealrichardson or @jonkeane who asked for the change due to a conflict on the R build.

---

I think it was a red herring, it hasn't been a problem over the years we've been building that R build, fine to revert.

---

Ah, I see.

In general, we use `.env` only for Docker Compose. And we don't use `.env` values when we run tests in Docker containers. We use `.env` values only when we build Docker images. So `.env` values will not conflict with environment variables for CI scripts including `r/tools/nixlibs.R` and `r/inst/build_arrow_static.sh`.